src/search/sr_optimizers.c

```
@@ -0,0 +1,333 @@
+}
+
+static int sr_optimizer_config_is_default(const sr_optimizer_config *cfg)
+{
+  if (!cfg) return 1;
+  if (cfg->max_evals > 0) return 0;
+  if (cfg->max_iters > 0) return 0;
+  if (cfg->seed != 0) return 0;
+  return 1;
+}
+
+static void sr_optimizer_log_int(FILE *output, const char *label, int value)
```

Clang (14.0.6)

[warning] parameter 'output' is unused

(misc-unused-parameters)